plugins/modules/site_workflow_manager.py

```diff
@@ -482,29 +567,126 @@ def site_exists(self):
           'get_site' function in the 'sites' family. It utilizes the
           'site_name' parameter from the 'want' attribute to identify the site.
         """
-
         site_exists = False
         current_site = {}
         response = None
+        if self.get_ccc_version_as_integer() <= self.version_2_3_5_3:
+            response = self.get_site_id(self.want.get("site_name"))
+            if response:
```

Comment:
Split it with the check if you not receive the response.
```
if not response:
....

Reply:
Added